### PR TITLE
Migrate retirement app into cf.gov repo

### PR DESCRIPTION
**Pull request by anselmbradford**
Migrates retirement app into cf.gov repo. Removes retirement django admin requirement. 

## Changes

- Migrate retirement app into cf.gov repo.

## How to test this PR

1. Compare http://localhost:8000/consumer-tools/retirement/before-you-claim/ to https://www.consumerfinance.gov/consumer-tools/retirement/before-you-claim/
2. Compare http://localhost:8000/consumer-tools/retirement/before-you-claim/es/ to https://www.consumerfinance.gov/consumer-tools/retirement/before-you-claim/es/
3. Compare http://localhost:8000/consumer-tools/retirement/before-you-claim/about/ to https://www.consumerfinance.gov/consumer-tools/retirement/before-you-claim/about/es/

## Notes and todos

- @higs4281 ~~Can `admin.py`, `migrations`, `models.py`,~~ (done) and maybe others be deleted? I left them in because I wasn't sure.

### CODE REVIEW

File: cfgov/retirement_api/data/mock_data/ssa_cola.html [hunk 1]
```diff
@@ -0,0 +1,619 @@
...
+<li><a class="btn-more" href="/socialmedia/" title="More Social Media"><svg focusable="false"><use xlink:href="/framework/images/icons/svg/sprite.svg#icon-plus"></use></svg></a></li>
+</ul>
+</div>
+
+</footer>
+<!-- END FOOTER -->
+</div><!-- end #page -->
+<!-- OCOMM BODY CONTENT -->
+<!-- DEFINITION PANEL -->
+<aside class="definition-panel" id="definition-panel" role="dialog"></aside>
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply cfgov/retirement_api/data/mock_data/ssa_cola.html:526]: mock data

File: cfgov/retirement_api/data/mock_data/ssa_awi_series.html [hunk 2]
```diff
@@ -0,0 +1,703 @@
...
+<li><a class="btn-more" href="/socialmedia/" title="More Social Media"><svg focusable="false"><use xlink:href="/framework/images/icons/svg/sprite.svg#icon-plus"></use></svg></a></li>
+</ul>
+</div>
+
+</footer>
+<!-- END FOOTER -->
+</div><!-- end #page -->
+<!-- OCOMM BODY CONTENT -->
+<!-- DEFINITION PANEL -->
+<aside class="definition-panel" id="definition-panel" role="dialog"></aside>
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply cfgov/retirement_api/data/mock_data/ssa_awi_series.html:610]: mock data

File: cfgov/retirement_api/data/mock_data/ssa_life.html [hunk 3]
```diff
@@ -0,0 +1,1649 @@
...
+</ul>
+</div>
+
+</footer>
+<!-- END FOOTER -->
+
+</div><!-- end #page -->
+<!-- OCOMM BODY CONTENT -->
+<!-- DEFINITION PANEL -->
+<aside class="definition-panel" id="definition-panel" role="dialog"></aside>
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply cfgov/retirement_api/data/mock_data/ssa_life.html:1556]: mock data

File: cfgov/retirement_api/data/mock_data/ssa_life.html [hunk 4]
```diff
@@ -0,0 +1,1649 @@
...
+
+<!-- DISCLAIMER DIALOG -->
+<aside class="dialog" id="disclaimer-dialog" role="dialog">
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply cfgov/retirement_api/data/mock_data/ssa_life.html:1559]: mock data

File: cfgov/retirement_api/data/mock_data/ssa_awi_series.html [hunk 5]
```diff
@@ -0,0 +1,703 @@
...
+
+<!-- DISCLAIMER DIALOG -->
+<aside class="dialog" id="disclaimer-dialog" role="dialog">
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply cfgov/retirement_api/data/mock_data/ssa_awi_series.html:613]: mock data

File: cfgov/retirement_api/data/mock_data/ssa_cola.html [hunk 6]
```diff
@@ -0,0 +1,619 @@
...
+
+<!-- DISCLAIMER DIALOG -->
+<aside class="dialog" id="disclaimer-dialog" role="dialog">
```
[the remark this answers is not in the document]
Comment (anselmbradford) [reply cfgov/retirement_api/data/mock_data/ssa_cola.html:529]: mock data

**Review by higs4281 — COMMENTED** [review]
This is looking very good. The tool performs as expected, and the content mostly matches, with most differences being in spacing and typeface weights, which I have no objection to.
 
I haven't delved into the Spanish version, but I found a couple small content glitches in the English version:

Content differences:

Step 2

Q1: Are you married?

MISSING LINK: Under answeer "Yes," the link to "Learn more about spousal benefits" is missing at the end.
MISSING LINK: Under answer "No," the link to " Learn more about your Social Security benefits after a divorce" is missing at the end.
MISSING LINK: Under answer "Widowed," the link to "Learn more about Social Security survivor benefits" is missing at the end.


Q4: Do you expect to have additional sources of retirement income beyond Social Security?

MISSIG TOOL TIPS: Under answer "Not sure," two tooltip references are missing.

**Comment by anselmbradford** [comment]
> Step 2
>
> Q1: Are you married?
> 
> MISSING LINK: Under answeer "Yes," the link to "Learn more about spousal benefits" is missing at the end.
> MISSING LINK: Under answer "No," the link to " Learn more about your Social Security benefits after a divorce" is missing at the end.
> MISSING LINK: Under answer "Widowed," the link to "Learn more about Social Security survivor benefits" is missing at the end.
>
> Q4: Do you expect to have additional sources of retirement income beyond Social Security?
> 
> MISSIG TOOL TIPS: Under answer "Not sure," two tooltip references are missing.

Fixed in https://github.com/cfpb/consumerfinance.gov/pull/6431/commits/73b21b6c2a55bf3300e139ded6c20b1fe8123ebd

Thanks @higs4281 !!!

**Comment by contolini** [comment]
The code coverage is below 100% which is causing the PR checks to fail. Improving the coverage is beyond the scope of this PR so no biggie but it looks like some of the [test files](https://github.com/cfpb/consumerfinance.gov/pull/6431/checks?check_run_id=2728319724#step:6:11) are being included in the coverage report. Is that expected? Should they be [omitted](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/.coveragerc)?

**Comment by higs4281** [comment]
There are some content chunks that aren't translated on the Spanish version on the PR stack. Three bits in the intro:
The main subhed, the Step 1 intro text, and the calculator explainer:

untranslated_intro



Also, the "By age 85" section isn't translating:

by_age_85

**Comment by higs4281** [comment]
Also Step 3's explainer, "This will not affect ..."

step_3_affect

**Comment by higs4281** [comment]
And Question 4's question text:

question_4_text

**Comment by higs4281** [comment]
Question 5's "No se" answer has a typo. "hasta" is missing its h.

Question_5_hasta

**Comment by higs4281** [comment]
In Step 3, choices 62-66 all have the same content, which is missing some translations:
- "which is earlier than ..."
- "If you start collecting at this age ..."
- √ "If your income, employment, or marital status ..."
Also, the legal disclaimer at the bottom isn't translated.

Step_3_choices_62-66

**Comment by higs4281** [comment]
Step 3 selections for 67, 68/69, and 70 also have untranslated bits.

For 67:  
- "which is your Social Security full retirement claiming age."
- "That means you'll receive ..."

Step_3_choice_67


For 68/69:
- "which is later than your Social Security full retirement claiming age."
- "This means you will get an increase ..."

Step_3_choices_68-69


And for 70:
- "which is your maximum Social Security benefit claiming age."
- "This means you will get your maximum possible ..." 

Step_3_choice_70


### Note 
All the "which is ..." clauses need a preceding "años, " in Spanish

I think that's it.

**Review by higs4281 — APPROVED** [review]
All fixed! Bravo.

**Pull request merged**